packtools/sps/utils/xml_utils.py

```diff
@@ -51,10 +51,18 @@ def node_plain_text(node):
     if node is None:
         return ""
 
-    # Remove o conteúdo de todos os nós <xref>
-    for xref in node.findall(".//xref"):
-        # Limpa todo o conteúdo do nó <xref>
-        xref.clear()
+    if remove_xref:
```

Comment:
@Rossi-Luciano mude a lógica. Para saber se remove ou não o elemento `xref` com ou sem conteúdo, verifique o valor de `@ref-type`. Se o valor é `fn`, `and not content.isalpha()`: remover, caso contrário remover somente os elementos mantendo o conteúdo.